specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+          "type": "integer",
+          "minimum": 0,
+          "maximum": 30
+        },
+        "responseTimeout": {
+          "description": "Response timeout in seconds to wait for a command completion on a device. Defaults to 30 seconds.",
+          "type": "integer",
+          "minimum": 5,
+          "maximum": 30
+        },
+        "request": {
+          "description": "The payload for the device command."
+        },
+        "response": {
+          "description": "The payload of the device command response.",
+          "readOnly": true
+        },
+        "responseCode": {
+          "description": "The status code of the device command response.",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "DeviceCommandCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of device command executions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeviceCommand"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of device command executions.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DeviceCredentials": {
+      "type": "object",
+      "properties": {
+        "idScope": {
+          "description": "ID scope for connecting to the IoT Central application.",
+          "type": "string"
+        },
+        "symmetricKey": {
+          "description": "The symmetric key credentials for the device.",
+          "$ref": "#/definitions/SymmetricKey"
+        },
+        "x509": {
+          "description": "The X.509 credential information for the device.",
+          "$ref": "#/definitions/X509"
+        },
+        "tpm": {
+          "description": "The TPM credentials for the device.",
+          "$ref": "#/definitions/Tpm"
+        }
+      },
+      "required": [
+        "idScope"
+      ]
+    },
+    "DeviceProperties": {
+      "description": "Property values associated with the device.",
+      "type": "object",
+      "additionalProperties": true
+    },
+    "DeviceTelemetry": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The last known value of this device telemetry."
+        },
+        "timestamp": {
+          "description": "String-formatted date representing the time when the telemetry value was sent.",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "DeviceTemplate": {
+      "type": "object",
+      "properties": {
+        "@id": {
+          "description": "Unique ID of the device template.",
+          "type": "string",
+          "readOnly": true
+        },
+        "@type": {
+          "description": "The JSON-LD types of this device template.",
```

Comment:
True it is json-ld, but more specifically it is the Digital Twins Model Language. Shouldn't we mention DTML specifically?